saleor/static/js/components/variantPicker/VariantPicker.js

```diff
@@ -25,14 +25,23 @@ export default class VariantPicker extends Component {
     const { store, variants } = this.props
 
     const variant = variants.filter(v => !!Object.keys(v.attributes).length)[0]
-    const selection = variant ? variant.attributes : {}
 
+    const params = {}
+
+    location.search.substr(1).split("&").forEach((item) => {
+      let pair = item.split("=")
+      if (pair.length == 2) {
+        params[decodeURIComponent(pair[0])] = decodeURIComponent(pair[1])
+      }
+    })
```

Comment:
I'd recommend using [query-string](https://github.com/sindresorhus/query-string) for that purpose. We're using it in `CategoryPage.js` to store the state in URL.